### ISSUE TITLE
 zram kernel oops on PI4-8GB board in 32bit

### ISSUE THREAD

**Issue by markVnl** [NethServer/arm-dev#45]
Can not describe it better than here:

https://github.com/raspberrypi/linux/issues/3839#

seems to be fixed in kernel > 5,4,81 (we running 5.4.72 now)
https://github.com/raspberrypi/linux/commit/1bef5f25a69234613b92a0e2456870fee4a57efc

A waiting feedback from (CentOS) arm-dev:
https://lists.centos.org/pipermail/arm-dev/2020-December/007919.html

**Comment by markVnl** [comment]
Test-Build of kernel with these patches solves it:

https://github.com/markVnl/Centos-raspberrypi2-kernel/commit/f94b49b781e5a68ec16be31d4add706d3f178a48

```
# swapon --show
NAME       TYPE      SIZE USED PRIO
/dev/zram0 partition 3.7G   0B   -2

# free -m
              total        used        free      shared  buff/cache   available
Mem:           7876         399        6636          34         840        7363
Swap:          3755           0        3755

```

**Comment by markVnl** [comment]
RPI images 7.9.2009 RC1 and RC2 run the above mentioned home-build kernel, 

Centos is on de brink of releasing 5.4.84 for the RPI.
https://git.centos.org/rpms/raspberrypi2/c/9a366c2a8498487c8c75407ada5b6e016ea9a320?branch=c7-sig-altarch-lts-5-4

Ran this a (test) build of this to be reasled kernel too wihout the problem described here.

Once released, 5.4.84 will be an regular update a we get rid of the home-build kernel.
